projects/plugins/jetpack/extensions/index.json

```diff
@@ -64,7 +64,8 @@
 		"recipe",
 		"v6-video-frame-poster",
 		"videopress/video-chapters",
-		"paywall"
+		"paywall",
+		"ai-assistant-form-support"
```

Comment:
Not a big fan of the extension name, but it's temporary :-). I'd suggestion something like `jetpack-form-ai-assistant`